src/inbox/inbox.service.ts

```diff
@@ -69,9 +149,13 @@ export class InboxService {
       (message.isEncrypted() ? message.encryptedData : message.data).length <= this.config.getStorageEmbedMaxSize();
```

Comment:
The thumbnail size should also be taken into consideration when determining if message data and thumbnail should be embedded.